codegen-core/common-test-models/big-numbers.smithy

```diff
@@ -0,0 +1,143 @@
+$version: "2.0"
+
+namespace com.amazonaws.bignumbers
+
+use aws.protocols#restJson1
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+@restJson1
+service BigNumberService {
+    version: "2023-01-01"
+    operations: [ProcessBigNumbers]
+}
+
+@http(uri: "/process", method: "POST")
```

Comment:
there appears to be some code that handles `E` / scientific notation but I don't see any tests of that here